### DISCUSSION TITLE
[Plugin Request] Swift iOS?

### DISCUSSION THREAD

**Discussion by iangregsondev** [dotansimha/graphql-code-generator#3675] [General]
Hi.

Was wondering if there is a swift generator for producing types from graphql for use in Swift ?

Thanks

**Comment by dotansimha** [comment]
Hi @iangregsondev 
We currently don't have a plugin for it, mostly because we don't use Swift by ourselves, so implementing it for us will be difficult. 
If you wish, you can create your own plugin. PRs are welcome ;)
